src/vaadin-avatar.js

```diff
@@ -234,7 +249,7 @@ class AvatarElement extends ElementMixin(ThemableMixin(PolymerElement)) {
       this.__setTitle(this.name);
     }
 
-    this.setAttribute('role', 'button');
+    this.setAttribute('role', 'img');
```

Comment:
Ideally the element would respect it if the `role` attribute was set to something different from the outside.